src/openforms/variables/static_variables/static_variables.py

```diff
@@ -0,0 +1,16 @@
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from openforms.forms.constants import FormVariableDataTypes
```

Comment:
I think it makes more sense to move these variable-related constants to `openforms.variables`.

At this point, I think our app organization & dependencies is more looking like:

1. `openforms.forms` & `openforms.variables` on the same level
2. `openforms.submissions` makes use of forms & variables

All three can be considered "core" Open Forms. This is probably relevant for the documentation PR as well.